dev/tracing_tests/test/timeline_test.dart

```diff
@@ -63,6 +63,7 @@ void main() {
   test('Timeline', () async {
     // We don't have expectations around the first frame because there's a race around
     // the warm-up frame that we don't want to get involved in here.
+    debugProfileBuildsEnabledUserWidgets = false;
```

Comment:
this ends up making the comment just before this line feel out of place, maybe it should go before the comment?

Reply:
done